### ISSUE TITLE
Debugging is totally unberable with large buffers

### ISSUE THREAD

**Issue by pps83** [Microsoft/nodejstools#1572]
I use nodejs v6.10.2 on win10. Debugging with VS is totally unacceptable when you try to do something that's is not a "hello world" app.

For example, I load 10 zip files into `Buffer`s (each zip if 5MB). Then I create this simple object pseudo code):

```var X = {
  something:{a: 1, b: 2},
  else: {x:"abc", y: "999", z: "z"},
  files: {
    "file1.zip": {size: 5000000, data: <Buffer for file1.zip data>, stats: {some simple obj with stats}},
    "file1.zip": {size: 5000000, data: <Buffer for file2.zip data>, stats: {some simple obj with stats}},
    "file3.zip": ... etc
}
```

So, then if I try to inspect X in VS, it totally chokes, it pauses, waits ... waits and then fails, often VS crashes. Surprisingly, I didn't even try to inspect `data`, I simply tried to expand "file1.zip" without even trying to look into 5MB of `data`. 

Note, that VS 2017 has identical problem. It totally cannot handle this as well.

**Comment by pps83** [comment]
I simplified code, now I have a simple object where for each file name I create another simple object that has two fields `data` and 'file'. When I try to simply inspect object it takes loooong time with mouse cursor turning into a spinning wheel and then it fails as this:

![image](https://cloud.githubusercontent.com/assets/1614246/25885612/44b1269e-350e-11e7-983a-8aed66e3fba0.png)

From this point debugging is busted, VS is choking, constantly all kinds of popups showing up. Doing anything basically halts VS and it crashes shortly after.

FYI, I also tried NTVS.Dev.2017-05-09.VS.2015.msi with 64bit nodejs 6.10.3 (I have latest VS2013 up3). Note VS2017 exhibits absolutely identical miserable behavior.

In contrast if I inspect that in node-inspector it all works perfectly, with no issues whatsoever:

![image](https://cloud.githubusercontent.com/assets/1614246/25885697/d4ff1576-350e-11e7-8a0c-ad510d42f880.png)
